include/nanoeigenpy/utils/helpers.hpp

```diff
@@ -0,0 +1,26 @@
+/// Copyright 2025 INRIA
+
+#pragma once
+
+#include <nanobind/nanobind.h>
+
+namespace nanoeigenpy {
+namespace nb = nanobind;
+
+/*! * Symlink to the current scope the already registered class T.
+ *
+ * @tparam T The class type to be symlinked.
+ * @param m nanobind module.
+ *
+ * \returns true if the type T is effectively symlinked.
+ */
+template <typename T>
+inline bool register_symbolic_link_to_registered_type(nb::module_& m) {
```

Comment:
I never liked the name in eigenpy personally. I think we can come up with a better name, something like `check_registration_alias`?

What do you think @jorisv @jcarpent ?

Reply:
As discussed with @jcarpent, we can change the name later after declaring a deprecation.